src/index.js

```diff
@@ -1,6 +1,9 @@
-const fibonacci = (n) => {
-
+const fibonacci = (num) => {
+  let arrnum = [];
+  for (let i = 0; i < num; i++) {
+    i == 0 ? arrnum.push(1) : arrnum.push(arrnum[i - 2] + arrnum[i - 1] || 1);
```

Comment:
"ternary if" is used to assign, better use a common `if`, or change to:
```js
arrnum.push(i==0 ? 1 : arrnum[i - 2] + arrnum[i - 1] || 1 );
```
However, prioritize that your code is understandable over that it is short.

Reply:
Is It that hard to understand ? 
Anyway, thank you for  the suggestion,